app/views/layouts/application.html.erb

```diff
@@ -1,5 +1,15 @@
 <% content_for :body do %>
   <div class="smart_answer">
+    <% if @presenter && @presenter.start_node.survey_url(request.path) %>
```

Comment:
Sorry, just noticed this when testing locally today, please can we move the intervention banner below the breadcrumbs, this will keep it consistent with the position in other rendering apps.